### ISSUE TITLE
Facepile Keros issue - ARIA input fields have an accessible name

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#10877]
### Environment Information

- **Package version(s)**: 6.204.4
- **Browser**: Chrome

### Describe the issue:
Keros Fast pass reports an issue for Facepile component - 
ARIA input fields have an accessible name

Refer the below image:
![image](https://user-images.githubusercontent.com/20378231/66982522-edce5980-f0d3-11e9-8114-519540ea9503.png)

The issue can be found on - 
https://developer.microsoft.com/en-us/fabric#/controls/web/facepile

#### Actual behavior:
Keros reports an issue - ARIA input fields have an accessible name

#### Expected behavior:
There should be no issue reported by Keros

**Comment by marygans** [comment]
Hi @DhirajKhot thanks for submitting this issue, I was able to repro this. Aria labels are missing from the `ul` tag. My tool also flagged an aria-hidden issue - "ARIA hidden element must not contain focusable elements". Which is interesting that only one of these is a button. I will look into it more.
![Screen Shot 2019-10-17 at 9 41 25 AM](https://user-images.githubusercontent.com/38991459/67029533-59a7c680-f0c2-11e9-8294-2d5a2652e89c.png)
![Screen Shot 2019-10-17 at 9 41 18 AM](https://user-images.githubusercontent.com/38991459/67029534-59a7c680-f0c2-11e9-8dad-0dd3e51dfc1c.png)

I'll get in a fix for these, thanks.

**Comment by DikshaGargMS** [comment]
Hi @marygans, Is there any ETA for this work item?

**Comment by DikshaGargMS** [comment]
Thanks @marygans for fixing the issue. Along with your default version, can you please release it to version 6 as well? We are using 6.204.4 version currently and do not wish to move to version 7 immediately.
